common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -714,6 +717,13 @@ def _parse_video_xml(cls, xml, id_generator=None):
                 # for about a month we did it for Strings also).
                 field_data[attr] = deserialize_field(cls.fields[attr], value)
 
+        # Update the handout location with current course_id
+        if 'handout' in field_data.keys() and course_id:
+            handout_location = StaticContent.get_location_from_path(field_data['handout'])
+            if isinstance(handout_location, AssetLocator):
```

Comment:
when is a `handout_location` not an `AssetLocator`? And can you have a test for this case?